portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Contact Rokiatou Sissoko | Information Science Student</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <script src="script.js"></script>
+  </head>
+  <body> 
+    <div class="top">
+    <header>
+    <img src="images/mintgreen.jpg">
+    </header>
+
+    <nav>
+        <ul>
+            <!-- <li><a href="index.html">Home</a></li> -->
+            <li><a href="index.html">About Me</a></li>
+            <li><a href="works.html">Works</a></li>            
+            <li><a href="contact.html">Contact Me</a></li>
+        </ul>
+     </nav>
+     </div>
+     
+    <!-- 1. Add a list of projects you've worked on.
+     2. Add a link to your LinkedIn and GitHub profiles. -->
+     
+
+    <div class="main">
+      <div class="aboutflex">
+
+        <div class="abouttext">
+          <h1> Contact Me </h1>
+            <p> I aspire to take on projects that will impact millions of users globally. Feel free to reach out to me at rssissoko600@gmail.com for any opportunities or additional questions. </p>
+            
+            </br>
+            </br>
+
+            <div class="row">
+                <div class="column">
+                    <a href="https://www.linkedin.com/in/rokiatousissoko/"> 
+                        <img src="/images/linked.png" style="width:75%"/>
```

Comment:
Are these .png files included somewhere? I'm not seeing them ..

Reply:
Added the image files